controller_manager/src/ros2_control_node.cpp

```diff
@@ -48,11 +48,19 @@ int main(int argc, char ** argv)
     }
     RCLCPP_INFO(cm->get_logger(), "update rate is %d Hz", update_rate);
 
+
+    std::chrono::system_clock::time_point timepoint_start = std::chrono::system_clock::now();
```

Comment:
Why not to use `node->now()` method and type?

TBH, probably we should do the following: 
- use node->now() for absolute time (usually only used for publishing messages).
- use stady_clock for calculating `dt` to avoid "jumps" in time

@bmagyar , @Karsten1987 what do you think?

Reply:
it'd be very confusing to have two time-related measurements in an API that come from different time sources. whatever we do, they should originate from the same time